src/announcements/index.md

```diff
@@ -3,6 +3,10 @@ title: SAP Concur Developer Center - Announcements
 layout: reference
 ---
 
+2018-10-11 :: **Invoice API Release Notes Published**
```

Comment:
Are we sure we want to make it a habit of announcing release notes? Seems like it will drive down the value of these announcements due to noise.